src/Microsoft.AspNetCore.Identity.UI/Areas/Identity/Pages/Account/ExternalLogin.cshtml.cs

```diff
@@ -0,0 +1,137 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.ComponentModel.DataAnnotations;
+using System.Security.Claims;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.AspNetCore.Identity.UI.Pages.Account
+{
+    public class ExternalLoginModel : PageModel
```

Comment:
Is this code directly from the templates or does it need to be reviewed?

Reply:
From the templates